### PR TITLE
perf(logs): improve service logs perf and typings

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-805

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 937d87a0-2391-4f54-8dbb-e263f2b05c0e | [test-old-status](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/937d87a0-2391-4f54-8dbb-e263f2b05c0e/services/general) |
 /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Make it work, make it right, make it fast

File: apps/console/src/main.tsx [hunk 1]
```diff
@@ -95,10 +96,12 @@ root.render(
       <QueryClientProvider client={queryClient}>
         <Provider store={store}>
           <BrowserRouter>
-            <ModalProvider>
-              <App />
-              <ToastBehavior />
-            </ModalProvider>
+            <TooltipProvider>
```
Comment (RemiBonnet) [inline apps/console/src/main.tsx:99]: 👍

File: libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx [hunk 2]
```diff
@@ -98,12 +131,22 @@ export function PodLogs(props: PodLogsProps) {
     },
   ]
 
+  const podNameToColor = useMemo(() => {
```
Comment (RemiBonnet) [inline libs/pages/logs/environment/src/lib/ui/pod-logs/pod-logs.tsx:134]: 👍

File: libs/pages/logs/environment/src/lib/ui/row-pod/row-pod.tsx [hunk 3]
```diff
@@ -102,11 +74,15 @@ export function RowPod(props: RowPodProps) {
...
-          {dateFullFormat(data.created_at, utc ? 'UTC' : undefined, 'dd MMM, HH:mm:ss:SS')}
-        </div>
-        <div data-testid="cell-msg" className="select-text pr-6 pt-0.5 text-neutral-50 relative w-full">
-          <span className="whitespace-pre-wrap break-all">{convertToAnsi(data.message)}</span>
+          {dateFullFormat(data.created_at, utc ? 'UTC' : timeZone, 'dd MMM, HH:mm:ss:SS')}
         </div>
+        <Ansi
+          data-testid="cell-msg"
+          className="select-text pr-6 pt-0.5 text-neutral-50 relative w-full whitespace-pre-wrap break-all code-ansi"
+          linkify
```
Comment (RemiBonnet) [inline libs/pages/logs/environment/src/lib/ui/row-pod/row-pod.tsx:82]: Why you didn't use convertToAnsi that uniformize the usage of the Ansi package ?

**Review by RemiBonnet — APPROVED** [review]
Thanks @ctjhoa, I have just one question about ANSI helper

File: libs/pages/logs/environment/src/lib/ui/row-pod/row-pod.tsx (hunk 3 above)
Comment (ctjhoa) [reply libs/pages/logs/environment/src/lib/ui/row-pod/row-pod.tsx]: https://github.com/Qovery/console/pull/811/commits/c465c335df0b24a1bb94c415984287bd19ff0502

**Pull request merged**